hack/run-tests.sh

```diff
@@ -360,4 +362,12 @@ if ${CRDS_UPDATE}; then
   apply_crds ${deploy_folder}
 fi
 
+if [ "${KEEP_NAMESPACE}" = "false" ]; then
+  echo "-------- Pruning namespaces"
+  cd ${SCRIPT_DIR}/../test
```

Comment:
I need to make the test directory the work directory. We can avoid this by sending the history log file in the argument:
```sh
go run test/scripts/prune_namespaces.go test/logs/namespace_history.log
```
But I would need to tweak the GetNamespacesInHistory method a bit for doing so and I wanted to avoid to over-complicate things.
Also, I don't think this would be an issue when running the pipeline.

Reply:
it is ok like that